app/src/main/java/com/example/tour_a_versity_v3/ui/InfoActivity.kt

```diff
@@ -1,20 +1,19 @@
-package com.example.tour_a_versity_v3
+package com.example.tour_a_versity_v3.ui
 
 import android.content.Intent
 import android.os.Bundle
 import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
-import com.example.tour_a_versity_v3.CECH_Photos
 import com.example.tour_a_versity_v3.R
 
-class CECH_Info : AppCompatActivity() {
+class InfoActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity2)
 
         val button = findViewById<Button>(R.id.TucPhotoButton)
         button.setOnClickListener {
-            val intent = Intent(this, CECH_Photos::class.java)
+            val intent = Intent(this, PhotosActivity::class.java)
```

Comment:
* If you create a new activity class for each location, you will end up with a lot of classes with a lot of code duplication. So I recommend only having one activity for the Info page.
* From there you can use a DTO to populate the data and show the photos.
* I also recommend suffixing your class names so that they are more descriptive. `InfoActivity` clearly describes that this is an activity class.

Reply:
Yes... the class should not hard code location name, because then you'll need to change you program, every time you add/edit/remove a location.